test/alternator/test_metrics.py

```diff
@@ -208,6 +211,73 @@ def test_streams_operations(test_table_s, dynamodbstreams, metrics):
 # ListGlobalTables, UpdateGlobalTable, UpdateGlobalTableSettings,
 # DescribeLimits.
 
+###### Test metrics that are latency histograms of DynamoDB API operations:
+
+# Check that an operation sets the desired latency histograms to non-zero.
+# We don't know what this latency should be, but we can still check that
+# the desired latency metric gets updated.
+@contextmanager
+def check_sets_latency(metrics, operation_names):
+    the_metrics = get_metrics(metrics)
+    saved_latency_count = { x: get_metric(metrics, 'scylla_alternator_op_latency_count', {'op': x}, the_metrics) for x in operation_names }
```

Comment:
If I understand correctly, here you will not get a histogram the first time before there were any calls to the alternator. It's not super likely but if you only going to run one test it could happen

Reply:
Right. I guess I need to assume a saved count of zero if I can't find the relevant metric. I'll change it, but anyway I'll need to test this after your patch, which also changes other things.

Reply:
Turns out I don't need to change anything, my get_metric() function (test/alternator/test_metrics.py) already does this - if the metric is missing, 0.0 is returned and not an error. Basically, it works by starting with total=0.0 and then looks for metrics which have the right name and label, and sums them all up. Normally this will sum up all the metrics of the different shards, but if there are zero matches, it's also fine.

So I think my test will work after your patch.